docs/guides/deployment.md

```diff
@@ -6,9 +6,11 @@ platforms:
 - [Google Cloud Run](#cloud-run)
```

Comment:
I'd put a recommendation like: "We recommend Google Cloud Run or Hugging Face Spaces which have a free tier".